Src/Newtonsoft.Json.Tests/Properties/AssemblyInfo.cs

```diff
@@ -43,11 +45,14 @@
 [assembly: AssemblyTitle("Json.NET Tests .NET 3.5")]
 #elif NET40
 [assembly: AssemblyTitle("Json.NET Tests .NET 4.0")]
+[assembly: SecurityRules(SecurityRuleSet.Level1)]
 #else
-
 [assembly: AssemblyTitle("Json.NET Tests")]
+[assembly: SecurityRules(SecurityRuleSet.Level1)]
```

Comment:
This is the LOC that's causing the build issues. If it's present, we get the `System.IO.FileLoadException` complaining about the `SecurityRules` attribute; if I get rid of it, I get security transparency rule failures, but only under the DNXCore 5.0 test run. Not quite sure where to go from here.